components/general/OurValues.tsx

```diff
@@ -2,40 +2,28 @@ import ListItem from "./ItemList";
 import styles from "@/styles/OurValues.module.css";
 import { fontSyne } from "@/pages/_app";
 
-type ValueProps = {
+export interface IValue {
   isPhoneSize: boolean;
-};
+  listItems: IlistItems[];
+}
+
+interface IlistItems {
+  id: 1;
+  title: string;
+  content: string;
+  icon: string;
+  height: number;
+  width: number;
```

Comment:
This is a continuation of the previous code. Considering that the height and width are hard-coded on the page rather than passed in, it has been deleted